src/Resources/Resources/Az.Resources.psd1

```diff
@@ -65,7 +65,8 @@ RequiredAssemblies = '.\Microsoft.Azure.Management.Authorization.dll',
                '.\Microsoft.Extensions.DependencyInjection.Abstractions.dll', 
                '.\Microsoft.Extensions.Options.dll', 
                '.\Microsoft.Extensions.Primitives.dll', 
-               '.\Microsoft.Azure.Graph.RBAC.dll'
+               '.\Microsoft.Azure.Graph.RBAC.dll',
+               '.\System.Runtime.CompilerServices.Unsafe.dll'
```

Comment:
@matthchr out of curiosity, was the build failing without this?

Reply:
It was, yes